pylint/checkers/base.py

```diff
@@ -1872,6 +1894,11 @@ def visit_unaryop(self, node):
             operator, right = operand.ops[0]
             if operator not in self.reverse_op:
                 return
+            for _type in (_node_type(left), _node_type(right)):
+                if not _type:
+                    return
+                if isinstance(_type, astroid.Set):
```

Comment:
I'm talking about this kind of case:

```
>>> en('set(something)')
<Call() l.1 [] at 0x3e9b4f0>
>>> en('set(something)').inferred()
[<Instance of builtins.set at 0x65647408>]
>>>
```
The set call can't be inferred in this case because something is not defined or something we don't know how to process.